packages/bitcore-node/src/services/pruning.ts

```diff
@@ -34,10 +35,15 @@ export class PruningService {
   }
 
   async detectAndClear() {
-    for (let chainNetwork of Config.chainNetworks()) {
-      const { chain, network } = chainNetwork;
-      await this.processOldMempoolTxs(chain, network, MEMPOOL_AGE);
-      await this.processAllInvalidTxs(chain, network);
+    if (CHAIN && NETWORK) {
```

Comment:
it would be nice to check that CHAIN and NETWORK exists on Config.chainNetworks() to avoid silent errors.